GNUmakefile

```diff
@@ -0,0 +1,15 @@
+
+MDFILES := README.md $(wildcard core/*.md ecosystem/*.md drafts/*.md)
+HTMLFILES := $(MDFILES:.md=.html)
+
+all: $(HTMLFILES)
+clean:
+	rm -f $(HTMLFILES) *~ */*~
+.PHONY: all clean
+
+%.html: %.md
+	title=$$(sed -ne '20q; s/^Title: *//p;' $^); \
+	pagetitle=$${title:-$(notdir $*)}; \
+	pandoc -s -f gfm -t html -V "pagetitle:$$pagetitle" -o $@ \
```

Comment:
`markdown_github` has been deprecated in favor of `gfm`. Current pandoc version is 2.2.3.2 (1.16.0.2 is from Jan 2016: https://pandoc.org/releases.html#pandoc-1.16.0.2-12-jan-2016)

Reply:
It sounds like your pandoc may be out of date.  However, I just found out about cmark-gfm, so maybe before we merge this I should try to make it work with that.

Reply:
Okay, so looks like cmark-gfm is not a complete solution--it only outputs an HTML fragment, not a page.  On my version of pandoc (2.2.3), markdown_github is deprecated and html and html5 are synonymous.  So I could change to html5 format if it makes a difference with older versions, but it would be a bit of a bummer to have to use the old rendering engine.

Reply:
I am running a fully updated Ubuntu (Xenial = 16.04 LTS). I imagine that most people don't run Bionic as it was released at the end of April.

The version included with it is:
```
# pandoc --version
pandoc 1.16.0.2
Compiled with texmath 0.8.4.1, highlighting-kate 0.6.1.
```

I am not sure what the best solution is; maybe use different targets (unless auto-detect is viable?)?

Here are the formats supported by that version:
```
pandoc [OPTIONS] [FILES]
Input formats:  commonmark, docbook, docx, epub, haddock, html, json*, latex,
                markdown, markdown_github, markdown_mmd, markdown_phpextra,
                markdown_strict, mediawiki, native, odt, opml, org, rst, t2t,
                textile, twiki
                [ *only Pandoc's JSON version of native AST]
Output formats: asciidoc, beamer, commonmark, context, docbook, docx, dokuwiki,
                dzslides, epub, epub3, fb2, haddock, html, html5, icml, json*,
                latex, man, markdown, markdown_github, markdown_mmd,
                markdown_phpextra, markdown_strict, mediawiki, native, odt,
                opendocument, opml, org, pdf**, plain, revealjs, rst, rtf, s5,
                slideous, slidy, texinfo, textile
                [**for pdf output, use latex or beamer and -o FILENAME.pdf]
```